playbook.tex

```diff
@@ -0,0 +1,73 @@
+\section{Playbook}
+
+\begin{enumerate}
+  \item First, we need to open a Jira Ticket to attach our work and set the state to In Progress.
+  \item Clone or update your current version of both repositories in lsst github account lsst-control and lsst-puppet-hiera-private.
+  \item Create a branch on both repositories with the IT ticket number you created. \textit{(ex: IT-3956\_rke\_updates)}
+  \item Update the rke version to match the desired Kubernetes version.
+  \begin{enumerate}[label=(\alph*)]
+    \item Check the rke github repository for the release information \href{https://github.com/rancher/rke/releases}{rke releases}
```

Comment:
It might be helpful for others to say something here about the rke binary version string not being the same as the k8s version string.  It would also be useful to say something about getting the k8s version string from the gh "release" or running `rke config -l -a`.

Reply:
good idea, i had to figure that out myself.